src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -15,25 +25,71 @@ public class CachingInController {
      */
 
     @RequestMapping("/")
-    public String index() {
+    public String index(Model model) {
+        Cache cache = new Cache();
+        cache.setCacheID(11);
+        cache.setLatitude("90");
+        cache.setLongitude("100");
+        cache.setDescription("I am a cache :)");
+        model.addAttribute(cache);
```

Comment:
mock data to test endpoints and unit tests

Reply:
I'd keep mock data in a Service Stub class... avoid polluting a production controller with it.  Stubs are easier to swap out.